crates/wadm-types/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "wadm-types"
+description = "Types and validators for the wadm API"
+version = "0.11.2"
```

Comment:
It felt like these should probably be the same version as current wadm to start, but I could easily be swayed for this to be 0.1.0 as well

Reply:
I think 0.1.0 makes good sense, we may keep them aligned for a few releases but they'll quickly go out of sync